tests/phpunit/tests/user/retrievePassword.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Test retrieve_password, in wp-includes/user.php
+ *
+ * @group user
+ */
+class Tests_User_RetrievePassword extends WP_UnitTestCase {
+
```

Comment:
```suggestion
	protected $user;
```

There is planned [deprecation of dynamic properties](https://wiki.php.net/rfc/deprecate_dynamic_properties) in PHP 8.2. For this reason and for consistency with other tests, the `user` property should be declared.

Reply:
Done!